mapbox/app/src/main/java/com/mapbox/services/android/testapp/geocoding/GeocodingServiceActivity.java

```diff
@@ -122,10 +122,14 @@ private void updateValuesFromBundle(Bundle savedInstanceState) {
   }
 
   /**
-   * Builds Android location engine
+   * Builds location engine
    */
-  private synchronized void buildAndroidLocationEngine() {
```

Comment:
Why is `synchronized` no longer needed?

Reply:
@zugaldia 
Don't like to answering with a question but I'm pretty sure you have more context:
Why was `synchronized in the first place? Why `getLocationEngine()` in `AndroidLocationEngine`, `LostLocationEngine` and `GoogleLocationEngine` need to be `synchronized`?

BTW, I've tested it and I didn't find any issues so apparently is working as expected.

Reply:
@Guardiola31337 For thread safety, did your tests include that scenario (i.e. multi-threading)?

Reply:
@Guardiola31337 just seeing this is an `Activity`, I though this was part of the library, thanks for pointing that out. Nevermind.